src/main/java/uk/gov/hmcts/ccd/domain/service/casedeletion/TimeToLiveService.java

```diff
@@ -199,14 +215,18 @@ private TTL getTTLFromCaseData(Map<String, JsonNode> caseData) {
     }
 
     private TTL getTTLFromJson(JsonNode ttlJsonNode) {
-        if (ttlJsonNode != null) {
-            try {
-                return objectMapper.readValue(ttlJsonNode.toString(), TTL.class);
-            } catch (JsonProcessingException e) {
-                throw new ValidationException(FAILED_TO_READ_TTL_FROM_CASE_DATA);
-            }
+        if (isEmptyNode(ttlJsonNode)) {
+            ttlJsonNode = objectMapper.getNodeFactory().nullNode();
```

Comment:
Does this small change impact the existing behaviour on L108? I don't think we'll ever be able to reach that code anymore? Might want to revert the change to this method?